internal/pkg/util/fs/layout/manager.go

```diff
@@ -152,6 +159,20 @@ func (m *Manager) AddSymlink(path string, target string) error {
 	return nil
 }
 
+// overrideDir will substitute another directory to the one associated
+// to directory located by path
+func (m *Manager) overrideDir(path string, realpath string) {
+	m.ovDirs[path] = realpath
+}
+
+// GetOverridePath returns the real path for the session path
+func (m *Manager) GetOverridePath(path string) (string, error) {
+	if p, ok := m.ovDirs[path]; !ok {
+		return p, nil
+	}
+	return "", fmt.Errorf("no override directory %s", path)
```

Comment:
```suggestion
	return "", fmt.Errorf("no override directory: %s", path)
```

~All~ **most** of the error returns have the `:` right before a variable, for consistency maybe better to add it

Reply:
I don't think the result would be understandable in this case: `no override directory: /foo/bar`, it's better without the colon.

Reply:
@mem, I agree, it does sound a bit better in this case.